lib/BridgeAbstract.php

```diff
@@ -8,13 +8,26 @@
 	const MAINTAINER = 'No maintainer';
 	const CACHE_TIMEOUT = 3600;
 	const PARAMETERS = array();
+	const FORMATS = array(); // array of compatibles formats or null
```

Comment:
I removed this in this PR.
We should talk about potential feature in another PR.